conf/defaults.yaml

```diff
@@ -253,11 +247,16 @@ topology.trident.batch.emit.interval.millis: 500
 topology.testing.always.try.serialize: false
 topology.classpath: null
 topology.environment: null
-topology.bolts.outgoing.overflow.buffer.enable: false
-topology.disruptor.wait.timeout.millis: 1000
-topology.disruptor.batch.size: 100
-topology.disruptor.batch.timeout.millis: 1
-topology.disable.loadaware.messaging: false
+topology.bolts.outgoing.overflow.buffer.enable: false # TODO: Roshan : Whats this ?
+topology.disruptor.wait.timeout.millis: 1000  # TODO: Roshan: not used, but we may/not want this behavior
+topology.transfer.buffer.size: 50000
+topology.transfer.batch.size: 10
+topology.executor.receive.buffer.size: 50000
+topology.producer.batch.size: 1000  # TODO: Roshan:  rename
+topology.flush.tuple.freq.millis: 5000
+topology.spout.recvq.skips: 3  # Check recvQ once every N invocations of Spout's nextTuple() [when ACKs disabled]
```

Comment:
Yes it's a breaking change (for existence spouts) so need to address carefully. Was just a 2 cents since you added related configuration. Btw, JStorm supported 2 threads spout for a long time and seems like it's the default, so the approach sounds valid to me. (They recently dropped acker though)